rpc/types.go

```diff
@@ -0,0 +1,116 @@
+package rpc
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/NethermindEth/juno/core/felt"
+)
+
```

Comment:
Please separate these types into their own files so it is easier to quickly identify which file contains what.

Reply:
What should the separation be based on? Different endpoints might share same data types.